examples/console/Rakefile

```diff
@@ -1,33 +1,13 @@
-require 'rake/clean'
-require 'rspec/core/rake_task'
 require 'bundler/setup'
+require 'rspec/core/rake_task'
 
-directory 'lib/console'
-
-namespace :cargo do
-  task :build do
-    sh "cargo rustc --release -- -C link-args='-Wl,-undefined,dynamic_lookup'"
-  end
-
-  task :clean do
-    sh "cargo clean"
-  end
-end
-
-task :clobber => "cargo:clean"
-
-file "lib/console/native.bundle" => ["lib/console", "cargo:build"] do
-  cp "../../target/release/libconsole.dylib", "lib/console/native.bundle"
-end
-CLOBBER.include("lib/console/native.bundle")
-
-task :irb => "lib/console/native.bundle" do
-  exec "irb -Ilib -rconsole"
-end
+$example_name = 'console'
```

Comment:
@wycats suggests that you should use `Rake.application.options.example_name = 'console'` instead